### PR TITLE
apps/embed: remove embed code

### PR DESCRIPTION
**Pull request by fuzzylogic2000**
I hope, I found everything! But I have questions (added that to the comments, so the lines that are new are the questions). Could you have a look @phillimorland, @khamui  or @goapunk ?

### CODE REVIEW

File: adhocracy-plus/assets/js/popup-close.js
Comment (khamui) [inline adhocracy-plus/assets/js/popup-close.js:2]: yes, think this file can go

**Review by khamui — APPROVED** [review]
looking good to me! but will let someone with more proficiency about <kbd>embed</kbd> have a look :)

Question:
Should we leave the `docs/embed.md` and make a note there, that this feature has been removed? So in case someone is wondering where the embed went, it is at least documented there?
Hmm..but maybe this will be in the release notes anyways.

File: adhocracy-plus/assets/js/unload_warning.js
Comment (phillimorland) [inline adhocracy-plus/assets/js/unload_warning.js:29]: oooo so @goapunk un-commented it and I merged it but not sure how neither of us caught that, I think it was working after the vanilla refactor, does that sound right @goapunk?

**Review by phillimorland — APPROVED** [review]
awesome! I think there maybe some bits to remove in a4 too but guess we have to wait on mB there, can be merged after the file is deleted

File: adhocracy-plus/assets/js/unload_warning.js
Comment (goapunk) [reply adhocracy-plus/assets/js/unload_warning.js:29]: yes, I think we left it like this because it was working? or because we knew the embed was about to be removed ? :shrug:

**Comment by fuzzylogic2000** [comment]
Let's merge after the release...

**Comment by fuzzylogic2000** [comment]
@phillimorland @khamui I did a rebase. Could you have another look before merge?

**Review by phillimorland — APPROVED** [review]
Yey!

**Pull request merged**
